aten/src/ATen/native/README.md

```diff
@@ -74,10 +74,19 @@ signature.
 - `std::array<bool,N>` (where N is `1-4`).  NB: you MUST NOT put a space after the comma, otherwise
   this argument will not parse correctly.  (If you decide to fix this, make sure you fix the
   argument parser both in ATen and in PyTorch.)
+- `TensorOptions`.  Tensor options provide information about how a
+  tensor should be constructed; it is most useful when you are writing a
+  factory function, where you have no `Tensor` inputs and thus
+  cannot otherwise determine how to construct a `Tensor`.
 - `*` is a special sentinel argument, which doesn't translate into an actual
   argument, but indicates that in the Python bindings, any subsequent arguments
   must be specified as keyword arguments (and cannot be provided positionally).
 
+Functions with no tensor inputs are called **factory functions**, and
```

Comment:
Syntactic nit: Shouldn't "factory functions" be emphasized rather than bold? "... called *factory functions*" instead of "... called **factory functions**"